2024/calcite-react/demo/src/Components/View/View.tsx

```diff
@@ -0,0 +1,47 @@
+import { useEffect, useRef, useState } from "react"
+
+import WebMap from "@arcgis/core/WebMap";
+import MapView from "@arcgis/core/views/MapView";
+
+import "./View.scss";
+
+interface ViewProps {
+    webmap: string;
+}
+
+export const View = (props: ViewProps) => {
+    const viewRef = useRef(null) as React.RefObject<HTMLDivElement>;
```

Comment:
to reduce type assertions:

```suggestion
    const viewRef = useRef<HTMLDivElement | null>(null);
```